Lombiq.Tests.UI/Services/SqlServerManager.cs

```diff
@@ -16,13 +20,13 @@ public class SqlServerConfiguration
 
     /// <summary>
     /// Gets or sets the template to use to generate SQL Server connection strings. It needs to contain the <see
-    /// cref="DatabaseIdPlaceholder"/> placeholder in the database name so unique database names can be generated for
-    /// concurrently running UI tests.
+    /// cref="DatabaseIdPlaceholder"/> placeholder in the database name so unique database names can be generated
+    /// for concurrently running UI tests.
     /// </summary>
     public string ConnectionStringTemplate { get; set; } = TestConfigurationManager.GetConfiguration(
         "SqlServerDatabaseConfiguration:ConnectionStringTemplate",
         $"Server=.;Database=LombiqUITestingToolbox_{DatabaseIdPlaceholder};Integrated Security=True;" +
-            $"MultipleActiveResultSets=True;Connection Timeout=60;ConnectRetryCount=15;ConnectRetryInterval=5");
+        $"MultipleActiveResultSets=True;Connection Timeout=60;ConnectRetryCount=15;ConnectRetryInterval=5");
```

Comment:
This was intentional, so it's a different level than the parameters.

Reply:
It's not standard though.

Reply:
We frequently do such indentations.

Reply:
I guess not frequently enough cause I've never seen 'em.

Reply:
This is typical with `if`s with complex clauses, like (obviously if the lines are long):

```c#
if (bla &&
    (bla ||
        bla) &&
    bla)
{
```

Reply:
Yes, I have seen that kind, but these are different things. In your example there are parentheses so indentation is logical.

Reply:
And here we have the multiple parameters. The first line is a parameter, the second line is a parameter, the third line is... Part of the second parameter. You get the idea.

Reply:
![image](https://user-images.githubusercontent.com/4007293/168440575-6eaa038d-78ed-471e-aaf3-5b3299e55d8a.png)

Reply:
:D